h5py_wrapper/wrapper.py

```diff
@@ -83,6 +83,15 @@ def save(filename, d, write_mode='a', overwrite_dataset=False,
     else:
         try:
             if dict_label:
+                warnings.warn("Deprecated argument dict_label provided. "
+                              "`dict_label` will be removed in the next release. "
```

Comment:
we should be consistent with use of the backtick \`, either we use it around every variable name or around none.